resources/ory/values.yaml

```diff
@@ -1,30 +1,17 @@
 # Global or kyma related overrides
 global:
   domainName: "kyma.example.com"
-  postgresql:
-    postgresqlDatabase: db4hydra
-    postgresqlUsername: hydra
   istio:
     gateway:
       name: kyma-gateway
       namespace: kyma-system
   ory:
-    oathkeeper:
-      maester:
-        mode: sidecar
     hydra:
```

Comment:
Added the following comment:
> The following configuration is kept so ory reconciler will not fail the reconciliation. This can be removed after we remove https://github.com/kyma-incubator/reconciler/blob/00dd6cde0c036f502508dde74f8f90a2a677dff9/pkg/reconciler/instances/ory/action.go#L277